test/Table_Tests/src/Data_Formatter_Spec.enso

```diff
@@ -17,7 +17,6 @@ type Custom_Type_With_To_Text
     to_text self = "[CUSTOM = " + self.field.to_text + "]"
 
 type Custom_Type_With_Error
-    Custom_Type_With_Error_Data
 
     to_text : Text
     to_text self = Error.throw (Illegal_State_Error_Data "foo_error")
```

Comment:
I'm not sure if I understand what happened here? You removed the ctor and pass the Type in the tests, so that you want to rely on the static variant of `to_text` for formatting? I guess that can work, but shouldn't then `to_text` not have the `self` parameter anymore?

Reply:
probably being overzelaous, can be reverted